VTCare/client/src/components/NotesPopup.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <Transition>
+    <div class="modal-mask">
+      <div class="modal-wrapper">
+        <div class="modal-container">
+          <div class="info-container">
+            <div class="modal-header">
+              <a style="color:black"><i class="fa-solid fa-xmark fa-2x" @click="closeModal()"></i></a>
+            </div>
+            <div class="notes-header-body">
+              <div class="patient-info">
+                {{ patientName }}
+              </div>
+              <div class="Date">
+                {{formatDate(apptDate)}}
+              </div>
+            </div>
+          </div>
+          <div class="notes-container">
+              <div class="diagnosis-content">
+                  <div>
+                    <label for="diagnosis">Diagnosis</label>
+                  </div>
+                  <div class="textarea-parent">
+                    <textarea id="diagnosis" v-model="diagnosis"></textarea>
+                  </div>
+              </div>
+              <div class="prescription-content">
+                  <div>
+                    <label for="prescription">Prescription</label>
+                  </div>
+                  <div class="textarea-parent">
+                    <textarea id="prescription" v-model="prescription"></textarea>
+                  </div>
+              </div>
+              <div class="notes-content">
+                  <div>
+                    <label for="notes">Notes</label>
+                  </div>
+                  <div class="textarea-parent">
+                    <textarea id="notes" v-model="notes"></textarea>
+                  </div>
+              </div>
+              <div class="checkbox-submit">
+                <div class="checkbox">
+                  <input type="checkbox" id="signed" name="signed" @click="checked=!checked">&nbsp;
+                  <label for="signed">Ready to submit?</label>
+                </div>
+                <div class="button-container">
+                  <button :disabled="!checked" @click="persistAndClose" class="ready-button">
+                    submit
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+
+        </div>
+      </div>
+  </Transition>
+</template>
+
+<script>
```

Comment:
add lang="ts" to script, and wrap the exported object with DefineComponent(....)

Reply:
I noticed you've been adding that to a few files, but why? Is that just to be consistent everywhere?

Reply:
It causes the IDE and compiler to suggest better optimizations for language and is the preferred standard. Also, since it's been used everywhere, so it should be used here, and if either one of these is used without the other, the program crashes and build.